package.json

```diff
@@ -7,11 +7,11 @@
   },
   "description": "Update Docker Registry",
   "devDependencies": {
-    "@types/jest": "^28.1.8",
-    "@types/node": "^18.7.13",
-    "@typescript-eslint/parser": "^5.35.1",
-    "@vercel/ncc": "^0.34.0",
-    "eslint": "^8.22.0",
+    "@types/jest": "^26.0.15",
+    "@types/node": "^14.14.9",
```

Comment:
Why did you downgrade versions?

Reply:
This was from upstream, but I might have messed up the merge.